src/image_util.rs

```diff
@@ -41,7 +41,11 @@ pub fn load_from_path(path: &Path) -> ImgUtilResult<Vec<RgbaImage>> {
         .filter_map(|res| res.map_or(None, |e| Some(e.path())))
         .collect::<Vec<_>>();
 
-    files.sort();
```

Comment:
now I do wonder how that sorted in the way it did in your case :thinking:

Reply:
ok looking at your filenames for a second longer I realize that it is indeed caused by using a sort ordering that is not respecting numeric values as a whole but individual digits instead. \
It went 1, 10, 11, 12, ..., 19, 2, 20, 21, ..., 3, ...